pkg/server/server.go

```diff
@@ -383,31 +386,32 @@ func (s *Server) VerifyEmail(c *gin.Context, redirectURL string) {
 	if !exists {
 		s.redirectError(c, errors.ErrInvalidAccessToken)
 		// Redirecting user to UI if the user is not authorized
-		c.Redirect(http.StatusUnauthorized, redirectURL)
+		c.Redirect(http.StatusPermanentRedirect, redirectURL)
```

Comment:
What's the difference between s.redirect and c.Redirect? Why is it uppercase 'R' in c.Redirect and not lowercase?

Reply:
`c,Redirect` is a library function of `gin.Context` referred here as `c` and `s.redirect` is our function declared in this file. And `R` means function is exported and can be called by other files and `r` means function is not exported.

Reply:
Thanks for the explanation of R vs r.
Trying to still understand why we need to call both s.redirect and c.Redirect.

Reply:
Okay I thinkwe are getting confused here with the wrong name implied to `s.redirect` function. What this function actually does is encodes an object or a data as a JSON into the context before giving the response. Naming is not correct here for `redirect` function.